tests/mail/tests.py

```diff
@@ -1554,6 +1554,19 @@ def test_validate_multiline_headers(self):
                 "Subject\nMultiline", "Content", "from@example.com", ["to@example.com"]
             )
 
+    def test_mutate_after_send(self) -> None:
+        # Ticket #34904
+        #   - Changing email object after sending mutates mail in mail.outbox
```

Comment:
This comment is unnecessary, please remove.

Reply:
Ok, removed it in latest commit